### ISSUE TITLE
[Bug] Check for writable build directories fails for root dirs

### ISSUE THREAD

**Issue by GitMensch** [OpenCobolIDE/OpenCobolIDE#230]
This directory is writeable for all applications (including OCIDE which can save/save-as files without problems there). Does the problem roots in the double backslash?

~~~~~
---------------------------
Exception while compiling a file
---------------------------
An exception occured when compiling 'D:\\com.cob'.

Error=Build directory not writeable (D:\)
---------------------------
OK   
---------------------------
~~~~~

## Log
~~~~~
INFO::open_cobol_ide.compilers::compiling D:\com.cob
DEBUG::open_cobol_ide.compilers::dependencies of D:\com.cob: []
INFO::open_cobol_ide.controllers.cobol::running compilation thread: ['D:\\com.cob']
DEBUG::open_cobol_ide.compilers::detecting file type: D:\com.cob - cp1252
DEBUG::open_cobol_ide.compilers::file type: <FileType.EXECUTABLE: 0>
INFO::open_cobol_ide.compilers::compiling D:\com.cob
~~~~~

## System information

- OpenCobolIDE: 4.7.dev14
- GnuCOBOL: GNU Cobol 2.0.20140821
- Python: 3.4.3 (32bits)
- Qt: 5.5.0
- PyQt: 5.5
- pyqode.core: 2.7.0.dev33
- pyqode.cobol: 2.7.dev0
- pyqode.qt: 2.7.dev0
- pygments: 2.0.2
- QDarkStyle: 2.1

**Comment by ColinDuquesnoy** [comment]
I cannot reproduce this on my system, D:\test.cob compiles fine:

```
INFO::open_cobol_ide.compilers::compiling d:\test.cob
INFO::open_cobol_ide.compilers::command: D:\Documents\OpenCobolIDE\GnuCOBOL-Win32-MinGW\bin\cobc.exe -x -o d:\bin\test.exe -std=default -debug test.cob
DEBUG::open_cobol_ide.compilers::working directory: d:\
DEBUG::open_cobol_ide.compilers::system environment: ['PYTHONUNBUFFERED=1', 'VS100COMNTOOLS=C:\\Program Files (x86)\\Microsoft Visual Studio 10.0\\Common7\\Tools\\', 'TERM=xterm-color', 'COMMONPROGRAMW6432=C:\\Program Files\\Common Files', 'PROCESSOR_REVISION=1e05', 'COMPUTERNAME=DESKTOP-HIVPIU1', 'OS=Windows_NT', 'USERNAME=Colin', 'TCL_LIBRARY=D:\\Documents\\hackedit\\launchers\\dist\\tcl', 'PSMODULEPATH=C:\\Windows\\system32\\WindowsPowerShell\\v1.0\\Modules\\', 'SYSTEMDRIVE=C:', 'VS120COMNTOOLS=C:\\Program Files (x86)\\Microsoft Visual Studio 12.0\\Common7\\Tools\\', 'NUMBER_OF_PROCESSORS=4', 'TEMP=C:\\Users\\reneg\\AppData\\Local\\Temp', 'LOCALAPPDATA=C:\\Users\\reneg\\AppData\\Local', 'COB_CONFIG_DIR=C:\\OpenCobol\\config', 'PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 30 Stepping 5, GenuineIntel', 'GIT_PAGER=cat', 'CLICOLOR=1', 'COMMONPROGRAMFILES(X86)=C:\\Program Files (x86)\\Common Files', 'TMP=C:\\Users\\reneg\\AppData\\Local\\Temp', 'PROGRAMFILES(X86)=C:\\Program Files (x86)', 'WINDIR=C:\\Windows', 'QT_API=pyqt5', 'QT_LOGGING_TO_CONSOLE=1', 'LOGONSERVER=\\\\DESKTOP-HIVPIU1', 'SESSIONNAME=Console', 'ALLUSERSPROFILE=C:\\ProgramData', 'PROGRAMFILES=C:\\Program Files (x86)', 'TK_LIBRARY=D:\\Documents\\hackedit\\launchers\\dist\\tk', 'PROCESSOR_ARCHITEW6432=AMD64', 'CEMENT_FRAMEWORK_LOGGING=1', 'PROGRAMDATA=C:\\ProgramData', 'USERPROFILE=C:\\Users\\reneg', 'PUBLIC=C:\\Users\\Public', 'COB_COPY_DIR=C:\\OpenCobol\\copy', 'SYSTEMROOT=C:\\Windows', 'PROCESSOR_LEVEL=6', 'USERDOMAIN=DESKTOP-HIVPIU1', 'USERDOMAIN_ROAMINGPROFILE=DESKTOP-HIVPIU1', 'PROCESSOR_ARCHITECTURE=x86', 'COMMONPROGRAMFILES=C:\\Program Files (x86)\\Common Files', 'PROGRAMW6432=C:\\Program Files', 'HOMEDRIVE=C:', 'APPDATA=C:\\Users\\reneg\\AppData\\Roaming', 'PATH=C:\\OpenCobol;C:\\Python34\\;C:\\Python34\\Scripts;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\Program Files\\Microsoft SQL Server\\110\\Tools\\Binn\\;C:\\Program Files (x86)\\Microsoft SDKs\\TypeScript\\1.0\\;C:\\Program Files\\Microsoft SQL Server\\120\\Tools\\Binn\\;C:\\Users\\reneg\\AppData\\Local\\Programs\\Python\\Python35-32\\Scripts\\;C:\\Users\\reneg\\AppData\\Local\\Programs\\Python\\Python35-32\\', 'QT_PLUGIN_PATH=C:\\Python34\\lib\\site-packages\\PyQt5\\plugins', 'PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY', 'COMSPEC=C:\\Windows\\system32\\cmd.exe', 'HOMEPATH=\\Users\\reneg', 'PAGER=cat']
INFO::open_cobol_ide.compilers::compiler process exit code: 0
INFO::open_cobol_ide.compilers::compiler process output: 
INFO::open_cobol_ide.compilers::binary file (d:\bin\test.exe) created:  True
```

To check if the directory is writeable, I try to create a dummy file (filename="test") and see if I get a PermissionError. 

I've added a log message with the full traceback of the error, maybe this will help. I've also changed the dummy file name to "test_write_access.ocide".

**Comment by GitMensch** [comment]
The filename "test" was the problem as I had a folder D:\test.
You can add the bug to the list of solved issues in 4.7 preview 2.
